pootle/settings/50-project.conf

```diff
@@ -127,3 +127,5 @@ POOTLE_EXPORT_VIEW_LIMIT = 10000
 # The banner will appear immediately if you set it to 0 and open a browsing page
 # with stale statistics data.
 POOTLE_STATS_REFRESH_ATTEMPTS_COUNT = 2
+
+POOTLE_TOP_CONTRIBUTORS_CHUNK_SIZE = 4
```

Comment:
I see the purpose of the setting although I fail to see how this is needed and/or is justified. I frankly don't believe we should add a configuration option for every single querying type for data we use in the UI unless it's backed up by evidence that shows it's effectively needed.